src/CommaSeparatedFloatParser.php

```diff
@@ -11,10 +11,10 @@ protected function parse($value)
         }
         $returnedIntArr = [];
         foreach (explode(',', $value) as $num) {
-            if (!is_numeric($num)) {
+            if (!is_numeric(trim($num))) {
```

Comment:
ahhh ok, I forgot `IntParser` also checks for `is_numeric()`, I remembered just casting :)
OK good point then, removing it.